### PR TITLE
docs(angular): add note on the import.meta error in mf docs

### LINKED ISSUE #10711: Docs: mfe tutorial does not work with version 14.2.4
Hello

Following the steps of the below 

https://nx.dev/module-federation/dynamic-module-federation-with-angular

after login the navigation fails and the component is not shown

`nx serve dashboard --devRemotes=login`

with error

image

Thanks

EDIT: 14.1.9 + ng v13 has the same result

   Node : 18.0.0
   OS   : win32 x64
   npm  : 8.4.1

   nx : 14.2.4
   @nrwl/angular : 14.2.4
   @nrwl/cypress : 14.2.4
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.2.4
   @nrwl/eslint-plugin-nx : 14.2.4
   @nrwl/express : Not Found
   @nrwl/jest : 14.2.4
   @nrwl/js : Not Found
   @nrwl/linter : 14.2.4
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.2.4
   @nrwl/web : Not Found
   @nrwl/workspace : 14.2.4
   typescript : 4.7.3
   ---------------------------------------
   Community plugins:

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

Missing note about the import.meta error in docs

## Expected Behavior

Add the import.meta error as a note in the docs

## Related Issue(s)

Fixes #10711

### CODE REVIEW

File: docs/shared/guides/module-federation/dynamic-mfe-angular.md [hunk 1]
```diff
@@ -364,6 +364,12 @@ nx serve dashboard --devRemotes=login
 
 This concludes the setup required for a Micro Frontend approach using Static Module Federation.
 
+> _When serving module federation apps in dev mode locally, there'll be an error output to the console, `import.meta cannot be used outside of a module`, and the script that is coming from is `styles.js`. It's a known error output, but it doesn't actually cause any breakages from as far as our testing has shown. It's because Angular compiler attaches the styles.js file to the index.html in a script tag with defer._ >_It needs to be attached with `type=module`, but Angular can't make that change because it breaks HMR. They also provide no way of hooking into that process for us to patch it ourselves._
```
[the remark this answers is not in the document]
Comment (Coly010) [reply docs/shared/guides/module-federation/dynamic-mfe-angular.md:367]: looks like something prettier did when formatting.

[the remark this answers is not in the document]
Comment (Coly010) [reply docs/shared/guides/module-federation/dynamic-mfe-angular.md:367]: yeah it's prettier changing that 🤷

[the remark this answers is not in the document]
Comment (Coly010) [reply docs/shared/guides/module-federation/dynamic-mfe-angular.md]: Updated there to just move it to a new line

**Review by leosvelperez — APPROVED** [review]
LGTM! 🎉

**Pull request merged**